apollo-compiler/src/test/kotlin/com/apollographql/apollo3/compiler/ValidationTest.kt

```diff
@@ -5,54 +5,72 @@ import com.apollographql.apollo3.ast.parseAsGQLDocument
 import com.apollographql.apollo3.ast.validateAsExecutable
 import com.apollographql.apollo3.ast.validateAsSchemaAndAddApolloDefinition
 import com.apollographql.apollo3.compiler.TestUtils.checkExpected
+import com.apollographql.apollo3.compiler.TestUtils.findSchema
 import com.apollographql.apollo3.compiler.TestUtils.testParametersForGraphQLFilesIn
 import okio.buffer
 import okio.source
 import org.junit.Test
+import org.junit.experimental.runners.Enclosed
 import org.junit.runner.RunWith
 import org.junit.runners.Parameterized
 import java.io.File
+import kotlin.test.assertTrue
 
 @Suppress("UNUSED_PARAMETER")
-@RunWith(Parameterized::class)
-class ValidationTest(name: String, private val graphQLFile: File) {
-  private val separator = "\n------------\n"
+@RunWith(Enclosed::class)
```

Comment:
Didn't know about `Enclosed`, but at least on my side the new test doesn't seem to run when running `./gradlew :apollo-compiler:test`. Maybe put it in its own file instead?

Reply:
My bad, I ran the validation test from my IDE but didn't try from command line.

Reply:
For the record (not for this PR), we could move the test to https://github.com/google/TestParameterInjector. It's pretty handy (but not multiplatform)